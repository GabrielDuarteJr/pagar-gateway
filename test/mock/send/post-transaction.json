{
	"value": 3455,
	"description": "Casinha de cachorro",
	"paymentMethod": "debit_card",
	"cardNumber": "5353323183678885",
	"bearerName": "gabriel junior",
	"cardExpiration": "09/2021",
	"cvv": "362"
}
