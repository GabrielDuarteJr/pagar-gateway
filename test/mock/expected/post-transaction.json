{
  "value": "34.55",
  "description": "Casinha de cachorro",
  "paymentmethod": "debit_card",
  "cardnumber": "************8885",
  "bearername": "gabriel junior",
  "cardexpiration": "09/2021",
  "created_at": "2019-09-19T02:10:40.479Z"
}
